<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Input Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Settings File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Output Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_settings">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
